src/seedu/addressbook/AddressBook.java

```diff
@@ -312,7 +331,7 @@ private static void setupDefaultFileForStorage() {
      * and a valid file name as determined by {@link #hasValidFileName}.
      */
     private static boolean isValidFilePath(String filePath) {
-        if (filePath == null) {
+        if (noValidFilePath(filePath)) {
```

Comment:
Boolean functions should generally have a name starting with 'is'. Have a closer look at the java coding standard document.